src/datetime/mod.rs

```diff
@@ -79,11 +79,141 @@ pub enum SecondsFormat {
     __NonExhaustive,
 }
 
-/// ISO 8601 combined date and time with time zone.
+/// `DateTime` is the preferred type for working with real-world timestamps. It contains all the
+/// information needed to exactly specify a point in time, and to do calculations on it: a date,
```

Comment:
I'm not sure it is correct to say that `DateTime` "contains" a "timezone". It contains the offset, but not the timezone itself. Maybe that can be worded differently?

Reply:
I am not sure how to word it better. Changed it to "and associated time zone" for now. But the paragraph below explains that the time zone is tracked by the type system.

Reply:
As I explained, my beef is with the word "contains". The next paragraph goes into a lot more detail, so I think we should get this first, high-level description right.